packages/ra-ui-materialui/src/form/FormTab.tsx

```diff
@@ -35,7 +34,9 @@ export const FormTab = (props: FormTabProps) => {
 
     const renderContent = () => (
         <FormGroupContextProvider name={value.toString()}>
-            <span
+            <Stack
+                alignItems="flex-start"
+                sx={{ paddingLeft: 1, paddingRight: 1 }}
```

Comment:
Can users override this one? We usually use a styled root component with classes for the inner components

Reply:
`.RaTabbedForm .MuiStack`